### ISSUE TITLE
[Improvement]: support links for hotspots

### ISSUE THREAD

**Issue by passioneight** [pimcore/pimcore#12738]
### Improvement description

Would be great to support links (i.e., same as `pimcore_link`) for hotspots:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/57256630/180422776-8dd34132-c161-404d-ba02-b14062b1f1ae.png">

> Actually, same goes for all editables that are provided by Pimcore (e.g., WYSIWYG).

**Comment by passioneight** [comment]
@fashxp I managed to create an additional menu entry in the dropdown, but I'm stuck when it comes to instantiating the link in extjs. Any chance you can point my in the right direction?

**Comment by fashxp** [comment]
@passioneight Sorry, would need to investigate by myself. can't you take the asset type or as a blueprint.
Still interested to contribute?

**Comment by passioneight** [comment]
@fashxp sure, but it will be a while until I'll have some time.

Also, isn't there a new UI on the way? If so, maybe this change is no longer needed. Wdyt?

**Comment by fashxp** [comment]
yes it is, but that improvement is not only UI, right? ... and if we implement it, we probably would need to implement it for both UIs anyway.

**Comment by passioneight** [comment]
Ah yes, you are right. Don't know what I was thinking.
